pkg/destroy/aws/aws.go

```diff
@@ -539,6 +539,7 @@ func (search *iamRoleSearch) find(ctx context.Context) (arns []string, names []s
 		ctx,
 		&iam.ListRolesInput{},
 		func(results *iam.ListRolesOutput, lastPage bool) bool {
+			search.logger.Debugf("iterating over a page of %d IAM roles", len(results.Roles))
```

Comment:
These logger messages should be capitalized
```suggestion
			search.logger.Debugf("Iterating over a page of %d IAM roles", len(results.Roles))
```